test/unit/shared/util.test.ts

```diff
@@ -89,3 +90,15 @@ describe('shared/reformatError', (): void => {
   })
 
 })
+describe('shared/throwMojaloopFSPIOPError', (): void => {
```

Comment:
there is a need to refactor test for mkMojaloopFSPIOPError

Reply:
Ok done